default.nix

```diff
@@ -1,289 +1,28 @@
-{ nixpkgsFunc ? import ./nixpkgs
+{ pkgs ? import ./nixpkgs
 , system ? null
```

Comment:
I think this is clearer as 'nixpkgsFunc'; 'pkgs' and 'nixpkgs' are both commonly used in nix expressions to mean "the nixpkgs object", but this is in fact a different thing - the function that *generates* that object.

Also, this is not a refactor: this changes the exposed interface.